src/Commands/SendSalesCommand.php

```diff
@@ -252,8 +254,12 @@ private function notify(string $message, string $status): void
     {
         if ($email = config('trx_mall_upload_sales_data_api.notifications.mail.email')) {
             $name = config('trx_mall_upload_sales_data_api.notifications.mail.name');
-            Notification::route('mail', $email)
-                ->notify(new TrxApiStatusNotification($name, $status, $message));
+
+            $isEnabledFailureNotificationsOnly = config('trx_mall_upload_sales_data_api.notifications.mail.enable_failure_notifications_only');
+
+            if (($isEnabledFailureNotificationsOnly && $status == 'error') || ! $isEnabledFailureNotificationsOnly) {
+                Notification::route('mail', $email)->notify(new TrxApiStatusNotification($name, $status, $message));
+            }
```

Comment:
You could put the following code at the beginning of this method.

```php
if ($status === 'error' && config('trx_mall_upload_sales_data_api.notifications.mail.enable_failure_notifications_only')) {
    return;
}
```

Reply:
Understood the simpler way.
but it wouldn't be below code ?

```php
if ($status !== 'error' && config('trx_mall_upload_sales_data_api.notifications.mail.enable_failure_notifications_only')) { 
     return; 
}

```

Reply:
Oh yeah, you're right.